dags/genie-nf-validate.py

```diff
@@ -0,0 +1,65 @@
+from datetime import datetime
+
+from airflow.decorators import dag, task
+from airflow.models.param import Param
+
+from dag_content.nextflow_tower_functions import create_and_open_tower_workspace
+
+dag_params = {
```

Comment:
This is awesome. A great first example of a configuratble dag for launching Nextflow Tower runs